src/Drawer/Drawer.js

```diff
@@ -140,7 +169,7 @@ export default class Drawer extends Component {
         <Paper
           elevation={docked ? 0 : elevation}
           square
-          className={classNames(classes.paper, paperClassName)}
+          className={classNames(classes.paper, classes[anchor], paperClassName)}
```

Comment:
Now, I would call that a *smart code*, I'm not sure it's better, but if that what you prefer, I'm in 👍 .